### PR TITLE
Updated getting started instructions

### PR DESCRIPTION
**Pull request by akosma**
This PR updates the instructions in the "Getting Started" guide following the new self-signup feature.

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `how-to`, `tutorial`, `reference`, `explanation`, `cicd`, `dependency`
      as they show up in the changelog
- [ ] Link this PR to related issues if applicable.

### CODE REVIEW

File: docs/modules/ROOT/pages/tutorials/getting-started.adoc [hunk 1]
```diff
@@ -4,15 +4,17 @@ This tutorial describes your first steps with APPUiO Cloud, from signup to your
...
 
-To create your APPUiO Cloud account, fill the form https://www.appuio.ch/en/offering/cloud/[in this page]. You will receive an e-mail inviting you to create a password for your newly created account.
+To create your APPUiO Cloud account, open the https://appuio.cloud/register[APPUiO Cloud registration] page. Create your account signing up with your current Google, GitHub, or Microsoft account. The video below shows the registration process.
+
+video::GwP172nGp1g[youtube]
 
 == Step 2: Login
 
-Login to the https://portal.appuio.cloud/[APPUiO Cloud Portal] following the instructions in your welcome e-mail. You will be asked to provide a password and setup Two-Factor Authentication (2FA) in your account.
+Login to the https://portal.appuio.cloud/[APPUiO Cloud Portal] using your newly created account. You will be asked to provide a password and setup Two-Factor Authentication (2FA) in your account.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/tutorials/getting-started.adoc:13]:
```suggestion
Login to the https://portal.appuio.cloud/[APPUiO Cloud Portal] using your newly created account.
```

No, no password. I am uncertain if 2FA is enforced.

File: docs/modules/ROOT/pages/tutorials/getting-started.adoc [hunk 2]
```diff
@@ -4,15 +4,17 @@ This tutorial describes your first steps with APPUiO Cloud, from signup to your
...
 
-== Step 3: Create an Organization
+== Step 3: Add Billing Information and Create an Organization
 
-The first thing you need to do in the https://portal.appuio.cloud/[APPUiO Cloud Portal] is to create an organization for yourself and your team.
+The first thing you need to do in the https://portal.appuio.cloud/[APPUiO Cloud Portal] is to add your billing information, and then to create an organization for yourself and your team.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/tutorials/getting-started.adoc:17]:
```suggestion
The first thing you need to do in the https://portal.appuio.cloud/[APPUiO Cloud Portal] is to add your https://portal.appuio.cloud/billingentities/$new?edit=y[billing information, and then to https://portal.appuio.cloud/organizations/$new[create an organization] for yourself and your team.
```

File: docs/modules/ROOT/pages/tutorials/getting-started.adoc [hunk 3]
```diff
@@ -4,15 +4,17 @@ This tutorial describes your first steps with APPUiO Cloud, from signup to your
...
 
 WARNING: If you don't choose a default organization, you won't be able to create new projects in the OpenShift web console later.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/tutorials/getting-started.adoc:19]:
```suggestion
WARNING: If you don't https://portal.appuio.cloud/user[choose a default organization], you won't be able to create new projects in the OpenShift web console later.
```

**Review by corvus-ch — CHANGES_REQUESTED** [review]
Missed a change on login. Also, I suggest adding some links for convenience.

**Comment by akosma** [comment]
Changes added, thanks!

File: docs/modules/ROOT/pages/tutorials/getting-started.adoc [hunk 4]
```diff
@@ -4,17 +4,19 @@ This tutorial describes your first steps with APPUiO Cloud, from signup to your
...
 == Step 2: Login
 
-Login to the https://portal.appuio.cloud/[APPUiO Cloud Portal] following the instructions in your welcome e-mail. You will be asked to provide a password and setup Two-Factor Authentication (2FA) in your account.
+Login to the https://portal.appuio.cloud/[APPUiO Cloud Portal] using your newly created account.
 
-== Step 3: Create an Organization
+== Step 3: Add Billing Information and Create an Organization
 
-The first thing you need to do in the https://portal.appuio.cloud/[APPUiO Cloud Portal] is to create an organization for yourself and your team.
+The first thing you need to do in the https://portal.appuio.cloud/[APPUiO Cloud Portal] is to add your https://portal.appuio.cloud/billingentities/$new?edit=y[billing information, and then to https://portal.appuio.cloud/organizations/$new[create an organization] for yourself and your team.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/tutorials/getting-started.adoc:17]:
```suggestion
The first thing you need to do in the https://portal.appuio.cloud/[APPUiO Cloud Portal] is to add your https://portal.appuio.cloud/billingentities/$new?edit=y[billing information], and then to https://portal.appuio.cloud/organizations/$new[create an organization] for yourself and your team.
```

**Review by corvus-ch — APPROVED** [review]
Apart from the missing `]` this is no looking good.

**Comment by akosma** [comment]
All good? Can we merge?

**Pull request merged**